src/result/mod.rs

```diff
@@ -31,3 +31,16 @@ pub type JsResult<'b, T> = NeonResult<Handle<'b, T>>;
 pub trait JsResultExt<'a, V: Value> {
     fn or_throw<'b, C: Context<'b>>(self, cx: &mut C) -> JsResult<'a, V>;
 }
+
+impl <'a, V, E> JsResultExt<'a, V> for Result<Handle<'a, V>, Handle<'a, E>>
```

Comment:
I don't quite understand the use case for this. If I wrap something in a `try_catch` it's to prevent it from throwing. If I want to do something like map on a result that might have thrown, I already have a `Result` that I can do `map()` on and it'll throw.

Is this for cases where you want to map and do something back in the JS VM? Like:

```rust
cx.try_catch(do_something)
  .and_then(|v| {
      ... cx ... // do stuff in the JS VM
  })
  .or_throw(cx)
```